node.gyp

```diff
@@ -614,7 +614,7 @@
                 '-Wl,--no-whole-archive',
               ],
             }],
-          ]		       
+          ]
```

Comment:
It might not matter, but why not update on line 22 above and in `common.gypi` as well?